htdocs/core/lib/files.lib.php

```diff
@@ -2799,20 +2799,20 @@ function dol_check_secure_access_document($modulepart, $original_file, $entity,
 				exit;
 			}
 
-			/*$perm = GETPOST('perm', 'aZ09');
-			 $subperm = GETPOST('subperm', 'aZ09');
-			 if ($perm || $subperm)
```

Comment:
no need to check for `$subperm` here, because the check is done once again inside the if clause + `$perm` is mandatory but `$subperm` is not